### DISCUSSION TITLE
Gitops like deployment pipeline

### DISCUSSION THREAD

**Discussion by Malopieds** [woodpecker-ci/woodpecker#4159] [Ask a Question] [answered]
Hi all, as anyone ever achieved this kind of setup with a git repo (on gitea for me) and a pipeline deploying containers? I'd like to do it using docker-compose file but I'm struggling with setting up volumes in pipeline. Even after checking allow deployment I still have the error:
```Insufficient privileges to use volumes```.
If anyone has more experience in this domain that would be appreciated! Everything I use is selfhosted in case that changes anything.

Thanks in advance !

**Comment by zc-devs** [comment, marked as answer]
> Insufficient privileges to use volumes

Maybe checking [trusted](https://woodpecker-ci.org/docs/usage/project-settings#trusted) helps you.
